### ISSUE TITLE
Clipboard gets opened when <dot> is pressed on custom completion Items

### ISSUE THREAD

**Issue by Pranomvignesh** [microsoft/monaco-editor#2317]
**monaco-editor version:** 0.21.2
**Browser:** Chrome
**OS:** Mac OS
**Playground code that reproduces the issue:**

```js
function createDependencyProposals(range) {
    // returning a static list of proposals, not even looking at the prefix (filtering is done by the Monaco editor),
    // here you could do a server side lookup
    return [
        {
            label: 'object',
            kind: monaco.languages.CompletionItemKind.Field,
            range: range
        },
        {
            label: 'object.app',
            kind: monaco.languages.CompletionItemKind.Field,
            range: range
        },
        {
            label: 'object.sample',
            kind: monaco.languages.CompletionItemKind.Field,
            range: range
        },
        {
            label: 'object.app.new',
            kind: monaco.languages.CompletionItemKind.Field,
            range: range
        }
    ];
}


monaco.languages.registerCompletionItemProvider('html', {
    triggerCharacters : ['{'],
    provideCompletionItems: function (model, position) {
        // find out if we are completing a property in the 'dependencies' object.
        var word = model.getWordUntilPosition(position);
        var range = {
            startLineNumber: position.lineNumber,
            endLineNumber: position.lineNumber,
            startColumn: word.startColumn,
            endColumn: word.endColumn
        };
        return {
            suggestions: createDependencyProposals(range)
        };
    }
});

monaco.editor.create(document.getElementById("container"), {
    value: "<html></html>",
    language: "html"
});
```

I have registered a CompletionItemProvider for `HTML` language, when i press `.` <dot> in the completion provider, monaco seems to access clipboard

https://user-images.githubusercontent.com/29809906/105462907-55029280-5cb5-11eb-8336-09c2e11624ed.mov

When i tried the second time , a blocking error message will be shown by the browser

<img width="1680" alt="Screen Shot 2021-01-22 at 1 30 31 PM" src="https://user-images.githubusercontent.com/29809906/105463724-7d3ec100-5cb6-11eb-9e04-8fc04997473b.png">

**Comment by Pranomvignesh** [comment]
Closing this issue as it is already been reported in https://github.com/microsoft/monaco-editor/issues/2307#issue-786449429

@alexdima  When can we expect a monaco-editor version with this fix. Is this fix present in the monaco-editor@0.21.3?

**Comment by alexdima** [comment]
:+1: Yes that is the purpose of `0.21.3`
